engine/lite/request-response/app/src/main/scala/pl/touk/nussknacker/engine/requestresponse/http/RequestResponseHttpApp.scala

```diff
@@ -25,13 +26,17 @@ object RequestResponseHttpApp extends Directives with FailFastCirceSupport with
 
   implicit private val materializer: Materializer = Materializer(system)
 
-  val metricRegistry = LiteEngineMetrics.prepareRegistry(config)
-
-  val requestResponseApp = new RequestResponseHttpApp(config, metricRegistry)
-
   val managementPort = Try(args(0).toInt).getOrElse(8070)
   val processesPort = Try(args(1).toInt).getOrElse(8080)
 
+  lazy val hostname = sys.env.getOrElse(
+    "HOSTNAME",
+    throw new IllegalStateException("HOSTNAME environment variable unavailable"))
```

Comment:
I'm afraid this will make running on windows a bit more difficult ;) but probably we can live with it...

Reply:
I forgot about this case - I've added some solution for that.